gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Thu Mar 09 11:46:20 MST 2017
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.1-all.zip
```

Comment:
After updating the gradle version in the root `build.gradle` file, make sure to run `./gradlew clean wrapper` which will update this project to use version 3.4.1.  Currently it's using 3.1